yjit/src/invariants.rs

```diff
@@ -169,12 +169,12 @@ pub extern "C" fn rb_yjit_cme_invalidate(callee_cme: *const rb_callable_method_e
         return;
     }
 
-    Invariants::get_instance().cme_validity.remove(&callee_cme).map(|blocks| {
+    if let Some(blocks) = Invariants::get_instance().cme_validity.remove(&callee_cme) {
```

Comment:
This makes the backtrace from the crash reporter shallower but otherwise it's equivalent.